pinot-tools/src/main/java/org/apache/pinot/tools/realtime/provisioning/MemoryEstimator.java

```diff
@@ -107,6 +129,15 @@ public MemoryEstimator(TableConfig tableConfig, File sampleCompletedSegment, int
     _tableDataDir.mkdir();
   }
 
+  public MemoryEstimator(TableConfig tableConfig, File dataCharacteristicsFile, File schemaFile, int ingestionRate,
```

Comment:
(nit) Add small javadoc please to highlight how this constructor is different from the other one